specification/network/resource-manager/Microsoft.Network/stable/2020-08-01/examples/VpnGatewayGet.json

```diff
@@ -138,6 +138,7 @@
               }
             ]
           },
+          "natRules": [],
```

Comment:
>natRules [](start = 11, length = 8)

What does this look like when it's empty? Are you returning object with only the `id` property or a full natRule resource body?

Reply:
It returns     "natRules": [], when its empty.
When there are NAT rules defined, we return full natRule resource body.